### ISSUE TITLE
Add ExchangeUserAccountControl to list of non-filterable parameters

### ISSUE THREAD

**Issue by JustinGrote** [MicrosoftDocs/office-docs-powershell#9592]
![image](https://user-images.githubusercontent.com/15258962/176044960-6cc26808-6143-4dac-ad17-c439348c3715.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7fedacb0-0fd3-a5b2-ea2f-c3bf61452afc
* Version Independent ID: eb9f1f1f-2bd3-be97-e4b4-a2c8250f5dc2
* Content: [Filters in the EXO V2 module](https://docs.microsoft.com/en-us/powershell/exchange/filters-v2?view=exchange-ps)
* Content Source: [exchange/docs-conceptual/filters-v2.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/exchange/docs-conceptual/filters-v2.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@JustinGrote, ExchangeUserAccountControl isn't available on Get-Recipient, either; it only works with:

- Get-Mailbox
- Get-MailUser
- Get-RemoteMailbox

It makes sense if you think about it; Get-Recipient has to work with all kinds of recipient types, but the concept of an associated account only makes sense for those aforementioned user recipient types.

**Comment by JustinGrote** [comment]
@chrisda  I'm aware of that, but the help breaks it out by cmdlet, so it should be listed in the get-mailbox section

![image](https://user-images.githubusercontent.com/15258962/176049396-79887f6b-c6b2-4d75-8847-fabbb69a7030.png)

**Comment by chrisda** [comment]
@JustinGrote, I'm not sure I follow.

Your picture shows ExchangeUserAccountControl not working with Get-EXORecipient (expcected, as we discussed), while it appears to work fine with Get-Mailbox (expected, since ExchangeUserAccountControl is not called out in the table). I tested it in my own E5 tenant and it works fine with both Get-Mailbox and Get-EXOMailbox.

I tried the 'forbidden' attribute DeletedItemFlags from the table on Get-EXOMailbox and it didn't work, but it worked with Get-Mailbox.

This result leads me to believe that the table is really stating the filterable attributes that work with the older remote PowerShell cmdlets that do not work with their Get-EXO\* equivalents. In other words, DeletedItemFlags works with Get-Mailbox but not Get-EXOMailbox.

I realize this makes the table look silly, and it definitely requires some rewording to make that point clear.

Comments?
